src/Libraries/RevitNodes/Elements/FamilyParameter.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Linq;
+using Autodesk.Revit.DB;
+using RevitServices.Transactions;
+
+namespace Revit.Elements
+{
+    public class FamilyParameter
+    {
+        internal Autodesk.Revit.DB.FamilyParameter InternalFamilyParameter
+        {
+            get;
+            set;
+        }
+
+        internal FamilyParameter(Autodesk.Revit.DB.FamilyParameter internalFamilyParameter)
+        {
+            this.InternalFamilyParameter = internalFamilyParameter;
+        }
+
+        #region Properties
+
+        /// <summary>
+        /// The name of the parameter.
+        /// </summary>
+        public string Name
+        {
+            get
+            {
+                return InternalFamilyParameter.Definition.Name;
+            }
+        }
```

Comment:
can also make this shorter with `=> InternalFamilyParameter.Definition.Name;` on same line as declaration

Reply:
if it still makes it a property (not a method or field), then i'd use this for most of these properties, as long as that expression after `=>` isn't too long